apps/adminer.sh

```diff
@@ -18,16 +16,52 @@ debug_mode
 # Check if root
 root_check
 
+# Check if adminer is already installed
+print_text_in_color "$ICyan" "Checking if Adminer is already installed..."
+if is_this_installed adminer
+then
+    choice=$(whiptail --radiolist "It seems like 'Adminer' is already installed.\nChoose what you want to do.\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+    "Uninstall Adminer" "" ON \
+    "Reinstall Adminer" "" OFF 3>&1 1>&2 2>&3)
+    
+    case "$choice" in
+        "Uninstall Adminer")
+            # Check that the script can see the external IP (apache fails otherwise)
+            check_external_ip
+            print_text_in_color "$ICyan" "Uninstalling Adminer and resetting all settings..."
+            a2disconf adminer.conf
+            rm $ADMINER_CONF
+            rm $ADMINERDIR/adminer.php
+            check_command apt purge adminer -y
+            if ! restart_webserver
+            then
+                msg_box "Apache2 could not restart...\nThe script will exit."
```

Comment:
If Apache stops, that's an hard error caused by this script. Better report it to $ISSUES so that we can fix it.

Reply:
Okay

Reply:
this is done.